addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -601,6 +601,17 @@ def _import_fill_invoice_form(self, journal, tree, invoice_form, qty_factor):
                 invl_logs = self._import_fill_invoice_line_form(journal, invl_el, invoice_form, invoice_line_form, qty_factor)
                 logs += invl_logs
 
+        # ==== total tax (to avoid rounding mismatch) ====
+
+        # TODO: test it works. Test also that it's working with all UBL files.
```

Comment:
I'd agree with that comment :)

Reply:
woops, I forgot to remove this :sweat_smile: